Elemental/Components/AeTabs/AeTab.razor

```diff
@@ -0,0 +1,46 @@
+﻿@inherits HtmlElement
+<div class="@_Classname">
+    @foreach(var page in _Tabs)
+    {
+        <AeButton OnClickEvent="() => _ActivePage = page" class="@(page == _ActivePage? "blue active" : "dark")" Disabled="@(page.Disable)">@( page.Title is not null? page.Title : !string.IsNullOrWhiteSpace(page.Text)? page.Text : page.Key)</AeButton>  
+    }
```

Comment:
Switch to a render fragment like we were talking about. 

Tricky part is finding a good way to incorporate the onclick callback - check out the `AeCollapse` how it uses a `span`. I'm open to improvements on that so if anything comes to mind let me know!